export const handler = async () => {
  throw new Error('My test error');
};
